File: src/main/kotlin/hanablive/model/dto/command/GameActionType.kt
package eelst.ilike.hanablive.model.dto.command

enum class GameActionType(val value: String) {
    CLUE("clue"),
    DISCARD("discard"),
    DRAW("draw"),
    PLAY("play"),
    STATUS("status"),
    STRIKE("strike"),
    TURN("turn");
}

File: src/external/hanablive/dto/instruction/game_action_type.rs
use serde::Deserialize;

#[derive(Debug, Clone, Copy, PartialEq, Eq, Deserialize)]
pub enum GameActionType {
    #[serde(rename = "clue")]
    Clue,
    #[serde(rename = "discard")]
    Discard,
    #[serde(rename = "draw")]
    Draw,
    #[serde(rename = "play")]
    Play,
    #[serde(rename = "status")]
    Status,
    #[serde(rename = "strike")]
    Strike,
    #[serde(rename = "turn")]
    Turn,
}
